src/commands/create/index.ts

```diff
@@ -120,7 +85,7 @@ export async function handler(argv: ReturnType<typeof builder>['argv']): Promise
         throw new Error(`could not find a template with type ${type}`)
     }
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    await createProject({ type, name: name!, local, template })
+    await createProject({ type, name: name!, template })
```

Comment:
it might be nice to also lint-fix the project right after creating it. some files might not be bundled due to `.gitignore`